### ISSUE TITLE
Inconsistent plotting results with PyVistaQT versus Jupyter (Trame) backends

### ISSUE THREAD

**Issue by robertsoakes** [pyvista/pyvistaqt#361]
Issue: plotting volume and mesh data in the same viewport results in misalignment for the `QTInteractor` in X and Z planes, but not for Jupyter (Trame) backend.

Example 1: plot in `QTInteractor`. _Bone models are shifted up and to the right._
![image](https://user-images.githubusercontent.com/3779876/232130443-f6786c19-b877-401b-a1cf-2bc623d5d6aa.png)

Example 2: plot in JupyterLab with the Trame server backend.
![image](https://user-images.githubusercontent.com/3779876/232130770-fca401b2-ca6d-4e01-af39-a45431eb285f.png)

_I've spent most of the last day trying to track down the issue, and can't seem to figure it out. The Trame backend seems to have some kind of unified coordinate system that isn't present in the `QTInteractor`, but I've been through both the PyVista and Trame code bases and it isn't clear to me what that would be._

**Comment by larsoner** [comment]
Can you check to see if `pv.Plotter` itself (rather than QtInteractor / BackgroundPlotter) also has this problem?

**Comment by robertsoakes** [comment]
@larsoner I'll try and stage a test for this tomorrow.

_I'll be honest, I'm not completely sure how, though. Is there a way to pass a `pv.Plotter` instance to QT -- I guess I could probably just do a screen capture. The Jupyter `server` backend seems to work, but volume rendering is not supported in the current client renderer, and I wasn't able to get the deprecated renderers to display anything._

I spent some additional time poking at things, and the issue seems to be partly due to PyVista ignoring data in the `vtkImageData` direction matrix. I wasn't able to figure out how to apply the rotation matrix to the image data, but the results for a unit matrix and the directional matrix look exactly the same.

This is what the Jupyter Trame backend plots without the direction matrix:
![image](https://user-images.githubusercontent.com/3779876/232179071-5906639a-7afa-4754-b170-b58142a2e88e.png)

After having poked more at the Trame backend, it looks like PyVista is just passing the underlying VTK classes through? (Trame is super cool, BTW. Really looking forward to seeing where it goes.)

**Comment by larsoner** [comment]
> I'll be honest, I'm not completely sure how, though. Is there a way to pass a pv.Plotter instance to QT

No, I'm suggesting you avoid Qt entirely for the `pv.Plotter` test. Just use that interface with whatever window it pops up and see if it has the same problem or not. It will help identify if it's really a Qt-specific problem or not.
